modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,143 @@
+
+To grant permissions for all the namespaces on the Argo CD cluster using the `*` wildcard pattern, configure the `ArgoCD` CR in the following manner:
++
+.Example
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd-all-namespaces
+spec:
+  sourceNamespaces:
+    - '*'
+----
+
+[id="fixed-issues-1-12-0_{context}"]
+== Fixed issues
+
+* Before this update, the notifications controller could not use built-in functions in notification templates to obtain information about applications due to an incorrectrepository server address. This update fixes the issue by initializing the notification controller to use the correct repository server address. link:https://issues.redhat.com/browse/GITOPS-2867[GITOPS-2867]
+
+* Before this update, when a user created a `Job` from a `CronJob` resource, an error is displayed. Now, users can create `Job` executions from a `CronJob` resource defined in a deployed Argo CD application. link:https://issues.redhat.com/browse/GITOPS-3617[GITOPS-3617]
+
+* Before this update, in some instance when applications are deleted, their application environment card would remain in a half-deleted state until the page was refreshed. Now, the application environment card will be removed from the UI automatically after the Argo CD application is deleted. link:https://issues.redhat.com/browse/GITOPS-2677[GITOPS-2677]
+
+* Before this update, when a user deleted apps and namespaces in Argo CD, in some cases, the {gitops-title} plugin would display an error message. This update fixes the issue by updating some components from the dynamic plugin SDK and provides better error handling. As a result, the error message is not displayed after deleting apps or namespaces. link:https://issues.redhat.com/browse/GITOPS-2746[GITOPS-2746]
+
+* Before this update, the password for admin user used to reset to the default password when it was updated by the Argo CD CLI or the Argo CD UI. This update fixes the issue and the user can now update the admin password and the password is no longer set to default. link:https://issues.redhat.com/browse/GITOPS-3581[GITOPS-3581]
+
+[id="known-issues-1-12-0_{context}"]
+== Known Issues
+
+* There is currently a known issue that changes the functionality of routing in the OpenShift Console Dynamic plugin, `dynamic-console-sdk`, that is used by the {gitops-title} Dynamic Plugin. This issue causes the horizontal navigation bar, which is used to switch between the Application Overview page and the Deployment History page for an application, on the Application Overview page to not function correctly.
++
+Workaround: To view the Deployment History of an application, use the *Deployment History* link on the application instead of the link on the horizontal navigation bar on the Application Overview page. This issue has been observed on all supported cluster versions that use the {gitops-title} Dynamic Plugin.
```

Comment:
```suggestion
Workaround: To view the **Deployment History** page of an application, use the *Deployment History* link on the application instead of the link on the horizontal navigation bar on the *Application Overview* page. This issue has been observed on all supported cluster versions that use the {gitops-title} Dynamic Plugin.
```

Reply:
Fixed.